input/pagecontent/capability-statements.md

```diff
@@ -1,10 +1,19 @@
 
 The following CapabilityStatements define the specific capabilities that different types of systems are expected to have in order to comply with this implementation guide. Systems conforming to this implementation guide are expected to declare conformance to one or more of the following capability statements.
 
+- [AU eRequesting Placer CapabilityStatement](CapabilityStatement-au-erequesting-placer.html)
+
+This CapabilityStatement describes the basic rules for the [AU eRequesting Placer actor](ActorDefinition-au-erequesting-actor-placer.html) that is responsible for initiating and creating diagnostic service requests. The complete list of FHIR profiles, RESTful operations, and search parameters supported by AU eRequesting placers are defined in this CapabilityStatement.
+
+- [AU eRequesting Filler CapabilityStatement](CapabilityStatement-au-erequesting-filler.html)
```

Comment:
Commenting here to get an inline comment. 

Now that Filler and Patient Access CapabilityStatements exist, Filler and Patient Access ActorDefinitions capabilities can be updated

Reply:
fixed.